cypress/e2e/annual-review/worker.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Emails sent on key dates
+
+  As a provider,
```

Comment:
for this one, there is a scenario for sending emails to providers: 

`And an email for the "START" key date is sent to eligible providers
`

It looks like the second story above isn't quite right - they look pretty much the same.  Did you mean something like:

```
As consular staff,
I should get emails before the Annual Review start date,
so that I can delay it if I need to

As consular staff,
I should get emails on the Annual Review start date,
so that I am made aware the providers will be removed from the system if they don't respond within 6 weeks

As a provider,
I should get emails on the annual review start date,
so that I can review and confirm my details are up to date
```

Reply:
Those suggestions are correct user stories. They describe why we have implemented it that way. My point being that what's currently there looks like it was copied from another feature.